RecoLocalMuon/RPCRecHit/plugins/CSCSegtoRPC.cc

```diff
@@ -45,11 +48,20 @@ std::unique_ptr<RPCRecHitCollection> CSCSegtoRPC::thePoints(const CSCSegmentColl
       segmentsInThisEventInTheEndcap++;
     }
 
+    float myTime = -9999.;
+    float myTimeErr = -9999.;
+    int myBx = -99;
+
     if (debug)
       std::cout << "CSC \t loop over all the CSCSegments " << std::endl;
     for (segment = allCSCSegments->begin(); segment != allCSCSegments->end(); ++segment) {
       CSCDetId CSCId = segment->cscDetId();
 
+      myTime = segment->time();
+      myBx = round(myTime / 25.);
```

Comment:
magic number 25 should be a constexpr. In fact, isn't this number something that should come from the conditions DB?

Reply:
It is the time difference between bunch crossings. It is now defined as const.